operator/pkg/lbipam/lbipam.go

```diff
@@ -71,6 +72,11 @@ func newLBIPAM(params LBIPAMParams) *LBIPAM {
 		lbClasses = append(lbClasses, "io.cilium/bgp-control-plane")
 	}
 
+	jobGroup := params.JobCollection.NewGroup(
+		job.WithLogger(params.Logger),
+		job.WithPprofLabels(pprof.Labels("cell", "lbipam")),
```

Comment:
Just an idea, would it be possible to inject this label from the Cell automatically?

Reply:
I did think about this. It could be possible but only if we integrate this more tightly with the hive package. Can't do it as a separate package since that would cause an import cycle.